### DISCUSSION TITLE
/docker-entrypoint.d/30-shlink-servers-json.sh: line 11: can't create /usr/share/nginx/html/servers.json: Permission denied after docker update

### DISCUSSION THREAD

**Discussion by Timsto** [shlinkio/discussions#18] [Help wanted] [answered]
### shlink-web-client version

latest

### How do you use shlink-web-client

https://app.shlink.io

### Summary

after update to the newest docker image (tag:latest) i got the following error massage during container startup: /docker-entrypoint.d/30-shlink-servers-json.sh: line 11: can't create /usr/share/nginx/html/servers.json: Permission denied   
any ideas?

**Comment by acelaya** [comment, marked as answer]
This has been solved already (see https://github.com/shlinkio/shlink-web-client/issues/1046), so you probably have an outdated docker image. Try pulling it.
